mypyc/primitives/str_ops.py

```diff
@@ -109,3 +109,20 @@
     return_type=object_rprimitive,
     c_function_name='CPyStr_GetSlice',
     error_kind=ERR_MAGIC)
+
+# str.replace(old, new)
+method_op(
+    name='replace',
+    arg_types=[str_rprimitive, str_rprimitive],
```

Comment:
Based on the signature here [https://docs.python.org/3/c-api/unicode.html#c.PyUnicode_Replace](https://docs.python.org/3/c-api/unicode.html#c.PyUnicode_Replace), I think each these `arg_types` might need one additional `str_primitive` to account for the "self" argument.

I noticed after compiling:

```
message = 'hello world'.replace('world', 'mars')
print(message)
```

in `build/opts.txt` we get:

```
...
L3:
    r5 = 'hello world'
    r6 = 'world'
    r7 = 'mars'
    r8 = 'replace'
    r9 = CPyObject_CallMethodObjArgs(r5, r8, r6, r7, 0)
...
```

(I think because the arg types don't quite align in the `mypyc/irbuild/ll_builder.py::matching_call_c` check and `mypyc/irbuild/ll_builder.py::translate_special_method_call` will return `None` instead of the op for `PyUnicode_Replace`). With an additional `str_primitive` arg, I think we will get the desired:

```
...
L3:
    r5 = 'hello world'
    r6 = 'world'
    r7 = 'mars'
    r8 = PyUnicode_Replace(r5, r6, r7, -1)
...
```
Does that make sense?